test/Datadog.Trace.ClrProfiler.IntegrationTests/TestHelper.cs

```diff
@@ -52,23 +52,12 @@ public static string GetRuntimeIdentifier()
 
         public static string GetSolutionDirectory()
         {
-            var directories = Environment.CurrentDirectory
-                                         .Split(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar)
-                                         .TakeWhile(d => !string.Equals(d, "test", StringComparison.InvariantCultureIgnoreCase))
-                                         .ToArray();
+            string currentDirectory = Environment.CurrentDirectory;
 
-            if (Environment.OSVersion.Platform == PlatformID.Win32NT && char.IsLetter(directories[0][0]) && directories[0][1] == ':')
-            {
-                // on Windows, add the root directory '\' after the drive letter to the first segment (e.g. "C:\")
-                directories[0] = directories[0] + Path.DirectorySeparatorChar;
-            }
-            else
-            {
-                // on other platforms, add the root directory '/' before the first segment
-                directories[0] = '/' + directories[0] + Path.DirectorySeparatorChar;
-            }
+            int index = currentDirectory.Replace('\\', '/')
```

Comment:
Nitpick:
```csharp
int index = currentDirectory.Replace(Path.AltDirectorySeparatorChar, Path.DirectorySeparatorChar)
    	                                        .LastIndexOf(Path.DirectorySeparatorChar + "test" + Path.DirectorySeparatorChar, StringComparison.InvariantCultureIgnoreCase);
```
Might be more reliable to some potential corner cases since it leverages the .NET constants for path separator and will avoid replacing "\" with "/" on Windows.